sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 4.11.0 (Unreleased)
+## 4.11.0 (2023-05-09)
 
 ### Features Added
 
-- Added new enum values to `SystemEventNames` related to Storage Tasks.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added new enum values to `SystemEventNames` related to Storage Tasks, Azure Communication Services and health care APIs.
```

Comment:
super extra baby nit: Is the "health care APIs" a specific library? If so, might be good to call it by the "official" name here.